contrib/electrumpersonalserver.service

```diff
@@ -0,0 +1,19 @@
+# Electrum Personal Server: systemd unit
+# /etc/systemd/system/electrumpersonalserver.service
+
+[Unit]
+Description=Electrum Personal Server
+After=bitcoind.service
+
+[Service]
+ExecStart=/usr/bin/python3 /your-installation-path/server.py  [optional arguments]
```

Comment:
1. The call to `python3` isn't required as the `server.py` script has a platform independent shebang line (`#! /usr/bin/env python3`).
2. Wouldn't you also require the EPS installation path as argument when run from `systemd`?
3. I believe you need to add a wait, because `bitcoind` takes some time to start and the RPC interface is available. I implemented this by adding: `ExecStartPre=/usr/bin/sleep 60`.

Reply:
1. In the past I had issues with calling a script directly and had to use stuff like `ExecStart=/bin/sh -c "/example/script"`, this is why I call python first. I tested it and it works reliably.
1. The way I installed EPS (see [here](https://github.com/Stadicus/guides/blob/master/raspibolt/raspibolt_64_electrum.md)) the program uses its own directory automatically. Do you think it should point to a different directory like `~/.eps`?
1. It's hard to say how long `bitcoind` needs to start. EPS runs nontheless and patiently waits for it to finish syncing. I would not include a forced wait, as it works without.